### PR TITLE
Upgrade Anydesk package to version 4.0.1

### PR DESCRIPTION
**Pull request by azazel75**
###### Motivation for this change

Upgrade the package to latest release

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/remote/anydesk/default.nix [hunk 1]
```diff
@@ -1,10 +1,10 @@
 { stdenv, fetchurl, makeWrapper, makeDesktopItem
 , atk, cairo, gdk_pixbuf, glib, gnome2, gtk2, libGLU_combined, pango, xorg
-, lsb-release }:
+, lsb-release, freetype, fontconfig, pangox_compat, polkit, polkit_gnome }:
 
 let
   sha256 = {
-    "x86_64-linux" = "0g19sac4j3m1nf400vn6qcww7prqg2p4k4zsj74i109kk1396aa2";
+    "x86_64-linux" = "08kdxsg9npb1nmlr2jyq7p238735kqkp7c5xckxn6rc4cp12n2y2";
     "i686-linux"   = "1dd4ai2pclav9g872xil3x67bxy32gvz9pb3w76383pcsdh5zh45";
```
Comment (Infinisil) [inline pkgs/applications/networking/remote/anydesk/default.nix]: You need to update this hash as well

Comment (azazel75) [reply pkgs/applications/networking/remote/anydesk/default.nix]: I've done it, thanks

**Comment by Infinisil** [comment]
@GrahamcOfBorg build anydesk pkgsi686Linux.anydesk

**Comment by Infinisil** [comment]
Please format your commit message according to the [Contribution Guidelines](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#submitting-changes), also linked to from the PR template.

You can also squash both commits into one

**Comment by azazel75** [comment]
@Infinisil done that, take a look

**Pull request merged**